src/app/cards/card-item/card-item.component.html

```diff
@@ -39,33 +39,33 @@
             [formGroup]="processingForm">
           <input type="hidden" formControlName="id">
           <div (click)="processingNameFocusIn()">
-              <label for="pia-edit-name">{{ 'homepage.cards.processing_name' | translate }}</label>
+              <label for="pia-edit-name[{{processing.id}}]">{{ 'homepage.cards.processing_name' | translate }}</label>
```

Comment:
Did you do that to keep unique IDs on the same rendered HTML page ?

Reply:
Sels answer : the commit message `fix unique id in processing cards`

Reply:
Yes, it's a bit hacky but some behaviours seem to be related to label and input, both connected by the id... I will not investigate for 3 hours for this kind of bugfix.